proto/sdk/alpha/alpha.proto

```diff
@@ -141,7 +141,7 @@ service SDK {
     // Returns OUT_OF_RANGE if the Count is out of range [0,Capacity].
     rpc UpdateCounter(UpdateCounterRequest) returns (Counter) {
       option (google.api.http) = {
-        patch: "/v1alpha1/{counterUpdateRequest.name=counterUpdateRequests/*}"
+        patch: "/v1alpha1/counters/{counterUpdateRequest.name}"
```

Comment:
I am very curious to know what `counterUpdateRequest.name` does 😄

Reply:
It pulls in the name of the Counter from the message CounterUpdateRequest.

Reply:
aaaah! Neat!